docs/guides/development/pack-maker-tools.md

```diff
@@ -47,10 +47,17 @@ The logs for your Pack are grouped by execution, in reverse chronological order
 
 <img src="../../../images/pmt_overview.png" srcset="../../../images/pmt_overview_2x.png 2x" class="screenshot" alt="Execution logs">
 
-A new execution is logged each time your Pack is run, for example when a formula is calculated or a button is pressed. When a sync table syncs you'll see an execution for the initial sync, and additional execution for each continuation of the sync. Additionally, while a user is working in the formula editor your Pack's formula may be run multiple times in order to generate a preview.
-
 Each execution has a brief description, if it was successful or failed, approximately when it was run, and what version of the Pack was run (if not the latest).
 
+A new execution is logged each time your Pack is run, for example when a formula is calculated or a button is pressed. When a sync table syncs you'll see an execution for the initial sync, and additional execution for each continuation of the sync.
+
+!!! info "Duplicate executions"
+    When looking at the logs you may see duplicate executions for the same formula. Some common reasons for this include:
+
+    - Using the formula editor, which runs formulas multiple times as you type to generate a preview of the result.
+    - The doc being open in multiple tabs.
```

Comment:
The more common case of this would be multiple concurrent users of the same doc, I would mention that. If you and I have the same doc open, both of our browsers will run the formula.